packages/runtime/src/next/components/tests/controls/link-control.test.tsx

```diff
@@ -59,10 +66,13 @@ describe('Page', () => {
     },
   )
 
-  describe('Link control data OPEN_PAGE', () => {
+  describe.each([null, 'fr-FR'])('Link control data OPEN_PAGE for locale %s', locale => {
+    const pathNameSlice = locale ? localizedPagePathnameSlice : pagePathnameSlice
```

Comment:
```suggestion
    const pathnameSlice = locale ? localizedPagePathnameSlice : pagePathnameSlice
```

Note, if you accept the suggestion above, you'd also need to update other references to `pathNameSlice` in this file